### DISCUSSION TITLE
Create a forum only for question and tutorials

### DISCUSSION THREAD

**Discussion by ninogjoni** [directus/directus#9710] [Feature Requests]
I see a lot of questions in the Discord group that are not answered and are lost or forgotten relatively quickly. In addition, the search for specific questions on Discord is unfortunately also cumbersome. My suggestion would be to create a forum for specific questions about the use of Directus with a forum tool such as [Discourse](https://www.discourse.org/). It would also be great to put tutorials e.g. how to integrate Directus in different environments.

What do you think?

**Comment by benhaynes** [comment]
I think this is a great idea... but is a big shift, so we should really think about it and consider all options before diving in. Let's compile a list of platforms that we could use for this for a bit, then we can start the transition. Also, we should think through if it makes sense to split this into _multiple_ platforms... such as a **forum** + **knowledgebase**.

* Discord
* Discourse
* GitHub Discussions
* HubSpot Knowledgebase
* [Knowledgebase.ai](https://www.knowledgebase.ai/)

I'm sure @rijkvanzanten and the rest of the @directus/team will have some great ideas here!

**Reply by rijkvanzanten** [reply]
I'm currently feeling like we might want to use GitHub Discussions exclusively for that purpose, as it seems to be very "Question & Answer" focussed (as proven by the big "Mark as answers" button. I basically see it as a scoped more friendly StackOverflow.

We'll have to find a better flow for feature requests, as GH Discussions just isn't great for it, after which we can make Discussions a little more focussed on what it's best at: Q&A

**Reply by 8byr0** [reply]
Totally follow on this point, github discussions is already a kind of forum! 
The main pain here is that it's not commonly used across open source communities so people don't know it / don't have reflex to start with it. Second pain is that discussions are not really highlighted when searching on google (or people may think they are issues).

Maybe a first step could be to improve communication around it and spread the word more than what's already done
